addons/account/tests/account_test_savepoint.py

```diff
@@ -55,7 +54,9 @@ def setup_company_data(cls, company_name, **kwargs):
         cls.env.user.company_ids |= company
         cls.env.user.company_id = company
 
-        chart_template = cls.env['account.chart.template'].browse(chart_template.id)
```

Comment:
Do you know why `browse` here? `chart_template` is already a recordset.

Reply:
no idea, I had the same thoughts

Reply:
@smetl any idea about this, and this PR in general?

Reply:
The results will always be different when running a localization. We don't expect the whole accounting tests are working for all localizations. I will add on master an additional parameter to specify the chart template when loading the setupclass, see: https://github.com/odoo/odoo/pull/41301/files#diff-f84d44ec52da269270420d56f27c3b72R617
You MUST not fallback on the generic_coa. We need to be able to use this setupclass in any localization separately to test additional features.

I don't remember why I'm rebrowsing the chart of account there.

Reply:
@rim-odoo see comment above since this has a major impact on odoo.sh.

Reply:
All of this in stable... I like to live dangerously too...

Reply:
> All of this in stable... I like to live dangerously too...

gné? The diff I'm linking will land on master...